AdobeReader.desktop

```
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=Adobe Reader 9
 MimeType=application/pdf;application/vnd.fdf;application/vnd.adobe.pdx;application/vnd.adobe.xdp+xml;application/vnd.adobe.xfdf;
-Exec=acroread 
+Exec=acroread %f
```

That should probably be `%F` (according to https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#exec-variables) because `acroread` can open multiple files at once.
```suggestion
Exec=acroread %F
```